src/metabase/api/common/validation.clj

```diff
@@ -20,7 +20,7 @@
 (defn check-embedding-enabled
   "Is embedding of Cards or Objects (secured access via `/api/embed` endpoints with a signed JWT enabled?"
   []
-  (api/check (embed.settings/enable-embedding)
+  (api/check (embed.settings/enable-embedding-static)
```

Comment:
so the `api/embed` endpoints are for static embedding only?

Reply:
That's right, we were using the general setting to gate those, but we can use this more accurate one now.

[ A static embed (or signed embed) is an iframe that’s loading a Metabase URL secured with a signed JSON Web Token (JWT).](https://www.metabase.com/docs/latest/embedding/static-embedding).